### PR TITLE
Bugfixes/verboseflag + Notes edge case (proper)

### PR DESCRIPTION
**Pull request by itfranck**
- Fix for multiple "Notes" field under different cases

- [Verbose implementation](https://github.com/PowerShell/SecretManagement/issues/66) + usage of `Write-Verbose` instead of `Write-Debug` since the latter is not supported.

### CODE REVIEW

File: SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1 [hunk 1]
```diff
@@ -85,6 +85,9 @@ function Get-Secret
 
     # TODO error handling
 
+    # -eq $true is used since the value will be null before Preview 5, which is equivalent to -verbose:$true
+    $VerboseFlag = @{Verbose = $AdditionalParameters.Verbose -eq $true }
```
Comment (TylerLeonhardt) [inline SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1]: What about:

```pwsh
if ($AdditionalParameters.Verbose) {
    $VerbosePreference = "Continue"
}
```

Then you shouldn't need the `@VerboseFlag` I don't think.

File: SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1 [hunk 2]
```diff
@@ -257,13 +265,16 @@ function Get-SecretInfo
...
+        $VerbosePreference = "Continue"
+    }
 
     $Filter = "*$Filter"
     $pattern = [WildcardPattern]::new($Filter)
     Invoke-lpass 'ls','-l' |
         Where-Object { 
             $IsMatch = $_ -match $lsLongOutput 
-            if (-not $IsMatch ) { Write-Debug -Message "No match for: $_ `nThis record will be ignored." }
+            if (-not $IsMatch ) { Write-Verbose -Message "No match for: $_ `nThis record will be ignored."  }
```
Comment (TylerLeonhardt) [inline SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1]:
```suggestion
            if (-not $IsMatch ) { Write-Verbose -Message "No match for: $_ `nThis record will be ignored." }
```

nit, extra space.

File: SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1 [hunk 3]
```diff
@@ -85,6 +85,10 @@ function Get-Secret
 
     # TODO error handling
 
+    if ($AdditionalParameters.Verbose) {
+    $VerbosePreference = "Continue"
```
Comment (TylerLeonhardt) [inline SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1]:
```suggestion
        $VerbosePreference = "Continue"
```
nit: need more space

File: SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1 (hunk 1 above)
Comment (itfranck) [reply SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1]: Indeed. Fixed.

**Review by TylerLeonhardt — APPROVED** [review]
LGTM with a couple whitespace nits

**Pull request merged**